changelogs/8.13.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[release-notes-8.13]]
+== APM version 8.13
+
+https://github.com/elastic/apm-server/compare/8.12\...8.13[View commits]
+
+* <<release-notes-8.13.0>>
+
+[float]
+[[release-notes-8.13.0]]
+=== APM version 8.13.0
+
+[float]
+==== Breaking Changes
+- Reverts a breaking change that ignored the beat name. This is visible in Stack Monitoring {pull}12176[12176]
+
+[float]
+==== Bug fixes
+
+[float]
+==== Deprecations
+
+[float]
+==== Intake API Changes
```

Comment:
could not find the way to remove these if no entries... rather than using something complicated I decided to keep them... so the reviewer will be the ones touching this